tests/system/action/base.py

```diff
@@ -196,3 +196,12 @@ def _fetch_groups(self, group_ids: List[int]) -> Dict[int, List[Dict[str, Any]]]
         for group in response.get(Collection("group"), {}).values():
             partitioned_groups[group["meeting_id"]].append(group)
         return partitioned_groups
+
+    def base_permission_test(
+        self, models: Dict[str, Any], action: str, action_data: Dict[str, Any]
+    ) -> None:
+        self.set_management_level(None)
+        self.set_models(models)
+        response = self.request(action, action_data)
+        self.assert_status_code(response, 403)
+        assert "You do not belong to meeting" in response.json["message"]
```

Comment:
I think it would be better to have a really permission error, i.e. `You are not allowed to perform action {self.name}`, see tests in openslide/openslides-backend/tests/system/action/test_permissions.py.
You could use the helper functions from openslide/openslides-backend/tests/system/action/base.py like create_meeting for a simple setup of base test

Reply:
A 403 is a real permission error. It is in the same way done, as Joshua did by tags.

Reply:
okay

Reply:
I agree with @r-peschke, see #609